.github/workflows/pullrequest.yml

```diff
@@ -136,7 +136,7 @@ jobs:
     timeout-minutes: 25
     env:
       AFFECTED_PROJECTS: ${{ matrix.projects }}
-      MAX_JOBS: 4
+      MAX_JOBS: 1
```

Comment:
Why this?

Reply:
To serialize the tests at the process level. I think we mentioned this during our meeting that seemed like some some tests were using a different, parallel running tests' DB. This is to exclude that possibility.

I have not measure this but it seems to me this parallelism was not buying us anything significant anyways.